test/connection-pool.test.ts

```diff
@@ -53,7 +53,9 @@ describe('connection pool', () => {
       .then(() => {
         throw new Error('expected to reject');
       })
-      .catch(err => expect(err).to.be.an.instanceof(GRPCConnectFailedError));
+      .catch((err) => {
+        expect(err).to.be.an.instanceof(GRPCGenericError);
```

Comment:
The change here is actually updating this mapping:

https://github.com/microsoft/etcd3/blob/5c638a68674cb530d0ac81b2e3dcb0e8c52eb8bf/src/errors.ts#L208-L246